test/contrib/test_securetransport.py

```diff
@@ -29,12 +29,15 @@ def teardown_module():
         pass
 
 
+# While SecureTransport supports TLS 1.3, we can't test for it because
```

Comment:
I now know that this comment is wrong: SecureTransport does not support TLS 1.3 at all

Reply:
I'll give it an update and merge it once CI passes on all code changes. :) Thanks @pquentin!